### ISSUE TITLE
Make URL path component available in some requests

### ISSUE THREAD

**Issue by plettich** [privacyidea/privacyidea#2019]
In some cases we receive data as part of the URL path (e.g. the `serial` in [`DELETE /token/<serial>`](https://github.com/privacyidea/privacyidea/blob/db90ab2129817aba7180e9e0777c785c656d8904/privacyidea/api/token.py#L551)). Flask does a nice job of extracting this data and passing it as a keyword parameter to the API function.
Currently our decorators (policies, events) do not use this data even though it is available in the wrapper function (e.g. in the [`event_wrapper()`](https://github.com/privacyidea/privacyidea/blob/2dd920e90ab42ed35fc717e29cec9c1e4c6b797a/privacyidea/lib/event.py#L59)).
We could extract possible data from the `**kwargs` parameter and pass it along to the event- and policy-objects.
TODO:
- [x] check all `before_after` functions for using the `view_args`

See also #2259

**Comment by cornelinux** [comment]
So you mean we could simply pass the ``kwds`` here (https://github.com/privacyidea/privacyidea/blob/master/privacyidea/lib/event.py#L75) and here (https://github.com/privacyidea/privacyidea/blob/master/privacyidea/lib/event.py#L93) - and also in the post section.

or should we concentrate on the ``serial`` and change it here:
https://github.com/privacyidea/privacyidea/blob/master/privacyidea/lib/event.py#L72

~~~~python
options = {"request": self.request,
                           "g": self.g,
                           "handler_def": e_handler_def,
                           "serial": kwds.get("serial")}
~~~~

I think we simply should pass all kwds to ``check_condition`` and to ``do`` in a first step (commit) and then let these implementation decide in a 2nd step, if and how keywords should be used.?

**Comment by cornelinux** [comment]
seems to be connected to #2535

**Comment by cornelinux** [comment]
@laclaro please check if this issue still persists or if #2535 actually also has solved this.

**Comment by plettich** [comment]
Fixed with #2535
